src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -315,14 +319,16 @@ private IContainer calculateGlobalPKAnalysisFor(Simulation simulation, Compound
          var vssPlasmaOverF = createParameter(VssPlasmaOverF, bloodPlasmaPK[Vss], VolumePerBodyWeight);
```

Comment:
@msevestre Do you remember, why all blood pk parameters are calculated either from peripheral venous blood or from venous blood (depending on the species), but **bioavailability is always calculated from the venous blood**?